server/lib/Room.js

```diff
@@ -8,6 +11,19 @@ const { v4: uuidv4 } = require('uuid');
 const jwt = require('jsonwebtoken');
 const userRoles = require('../userRoles');
 
+///////////////////////////////////// START /////////////////////////////////////////
+let mafaldaRouters
+let resourceUsages
+// @TODO: routers and transports are not used anywhere as of now.
```

Comment:
Please check this

Reply:
Routers not being used are ok, since MafaldaRouter abstract them. Transports are returned by the equivalent methods, and are being used directly.